indico/modules/events/abstracts/controllers/management.py

```diff
@@ -123,3 +140,63 @@ class RHAbstractListStaticURL(RHAbstractListBase):
 
     def _process(self):
         return jsonify(url=self.list_generator.generate_static_url())
+
+
+class RHCreateAbstract(RHAbstractListBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            del data['attachments']
```

Comment:
The latter, but it's pretty straightforward. You'd use a custom Field that uses the DropzoneWidget and then just gets the files from `valuelist` in `process_formdata` and puts them in `self.data`.